docs/topics/mta-web-reviewing-an-analysis-report.adoc

```diff
@@ -8,12 +8,13 @@
 
 An {ProductShortName} analysis report contains a variety of sections, including a listing of the technologies used by the application, the dependencies of the application, and the lines of code that must be changed to successfully migrate or modernize the application.
```

Comment:
@anarnold97 
This paragraph describes an analysis report, not an assessment report. And the heading of this section says "Reviewing an analysis report".  This seems to be inherited from the former section 6.6. And the "Reviewing an assessment report" section, I think, should be placed as a subsection of section 6.2, "Assessing applications", after 6.2.2. "Assessment Module". 
And the former section 6.6, "Reviewing an analysis report", should be after 6.8.2. Downloading an analysis report.